cg/models/orders/samples.py

```diff
@@ -0,0 +1,201 @@
+    formalin_fixation_time: Optional[int]
+    post_formalin_fixation_time: Optional[int]
+    tissue_block_size: Optional[str]
+    # "Not Required"
+    cohorts: Optional[List[str]]
+    phenotype_groups: Optional[List[str]]
+    phenotype_terms: Optional[List[str]]
+    quantity: Optional[int]
+    subject_id: Optional[
+        constr(
+            regex=NAME_PATTERN, max_length=models.Sample.subject_id.property.columns[0].type.length
+        )
+    ]
+    synopsis: Optional[str]
+
+
+class MipDnaSample(Of1508Sample):
+    # "Required if data analysis in Scout or vcf delivery"
+    panels: List[constr(max_length=models.Panel.abbrev.property.columns[0].type.length)]
+    status: StatusEnum
+
+
+class BalsamicSample(Of1508Sample):
+    pass
+
+
+class MipRnaSample(Of1508Sample):
+    time_point: Optional[NonNegativeInt]
+
+
+class FastqSample(BaseSample):
+    # Orderform 1508
+    # "required"
+    container: Optional[ContainerEnum]
+    sex: SexEnum = SexEnum.unknown
+    source: str
+    tumour: bool
+    # "required if plate"
+    container_name: Optional[str]
+    well_position: Optional[str]
+    elution_buffer: str
+    # "Not Required"
+    quantity: Optional[int]
+
+
+class RmlSample(BaseSample):
+    # 1604 Orderform Ready made libraries (RML)
+    # Order portal specific
+    # "This information is required"
+    pool: constr(max_length=models.Pool.name.property.columns[0].type.length)
+    concentration: float
+    concentration_sample: Optional[float]
+    index: str
+    index_number: Optional[str]
+    # "Required if Plate"
+    rml_plate_name: Optional[str]
+    well_position_rml: Optional[str]
+    # "Automatically generated (if not custom) or custom"
+    index_sequence: Optional[str]
+    # "Not required"
+    control: Optional[str]
+
+
+class MetagenomeSample(BaseSample):
+    # 1605 Orderform Microbial Metagenomes- 16S
+    # "This information is required"
+    container: Optional[ContainerEnum]
+    elution_buffer: str
+    source: str
+    # "Required if Plate"
+    container_name: Optional[str]
+    well_position: Optional[str]
+    # "This information is not required"
+    concentration_sample: Optional[float]
+    quantity: Optional[int]
+    extraction_method: Optional[str]
+
+
+class MicrobialSample(BaseSample):
+    # 1603 Orderform Microbial WGS
+    # "These fields are required"
+    organism: str
+    reference_genome: constr(
+        max_length=models.Sample.reference_genome.property.columns[0].type.length
+    )
+    elution_buffer: str
+    extraction_method: Optional[str]
+    container: ContainerEnum
+    # "Required if Plate"
+    container_name: Optional[str]
+    well_position: Optional[str]
+    # "Required if "Other" is chosen in column "Species""
+    organism_other: Optional[str]
+    # "These fields are not required"
+    concentration_sample: Optional[float]
+    quantity: Optional[int]
+    verified_organism: Optional[bool]  # sent to LIMS
+
+
+class MicrosaltSample(MicrobialSample):
```

Comment:
is this needed?

Reply:
I think it makes it a bit clearer since the Microsalt-samples and Sarscov2-samples use different orderforms but at this point shares a common base